CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ _Please add entries here for your pull requests that are not yet released._
 - Include turbolinks in dev and update webdrivers #1174
 - Add support for multiple `require.context` with addition of `useContexts` #1144
 - Update dependencies
+- Add `--function` and `--func` options for generating a React function component #1271
```

Comment:
Let's make this the default.

We can target this for 3.0.

Reply:
Done